tests/testthat/test_RFunction.R

```diff
@@ -1,13 +1,12 @@
-library('move2')
-
-test_data <- test_data("input3_move2.rds") #file must be move2!
+test_data <- test_data("input3_move2loc_LatLon.rds")
 
 test_that("happy path", {
-  actual <- rFunction(data = test_data, sdk = "unit test", year = 2005)
-  expect_equal(unique(lubridate::year(mt_time(actual))), 2005)
+  actual <- rFunction(data = test_data, sdk = "unit test", year = 1998)
+  # TODO: how to select the column of timestamps in a move2 object?
+  expect_equal(unique(lubridate::year(actual@timestamps)), 1998)
```

Comment:
Hallo @annescharf - kannst du mir helfen diesen Test zu reparieren? 

Diese `actual@timestamps` Notation ist noch von `move1`. Wie kann ich das mit `move2` bewerkstelligen?

Fehler: `Error in `actual@timestamps`: no applicable method for `@` applied to an object of class "move2"`

Reply:
HI @cleemansen die letzte Zeile muss sein:
expect_equal(unique(lubridate::year(mt_time(actual))), 1998)

Reply:
Vielen Danke, hat direkt geklappt..